python-main.js

```diff
@@ -1493,7 +1493,10 @@ function web_editor(config) {
 
         if (usePartialFlashing) {
             // Push binary to board
-            p = PartialFlashing.connectDapAsync()
+            p = doDisconnect()
+                .then(function() {
+                        return doConnect();
+                })
```

Comment:
@microbit-sam 
I just noticed that because of this disconnect->connect we are getting a connect event every time we flash. 
Is there anyway to solve that?

Reply:
Yeah, also I guess the UI will be changing

I can pull out the relevant parts of doDisconnect() and doConnect() into a seperate shared function, or if it's just a couple of commands I'll add them here